gensim/models/phrases.py

```diff
@@ -237,9 +220,8 @@ class Phrases(SentenceAnalyzer, PhrasesTransformation):
 
     def __init__(self, sentences=None, min_count=5, threshold=10.0,
                  max_vocab_size=40000000, delimiter=b'_', progress_per=10000,
-                 scoring='default', common_terms=frozenset()):
+                 scoring='default', common_terms=frozenset(), doc2vec=False):
```

Comment:
In general, this idea is bad: Phrases shouldn't depends on `doc2vec` or any other model